inotify/adapters.py

```diff
@@ -48,17 +50,17 @@ def __init__(self, type_name, event):
 
 
 class Inotify(object):
-    def __init__(self, paths=[], block_duration_s=_DEFAULT_EPOLL_BLOCK_DURATION_S):
-        self.__block_duration = block_duration_s
+    def __init__(self, paths=[], block_duration_ms=_DEFAULT_POLL_BLOCK_DURATION_MILLISECONDS):
+        self.__block_duration = block_duration_ms
         self.__watches = {}
         self.__watches_r = {}
         self.__buffer = b''
 
         self.__inotify_fd = inotify.calls.inotify_init()
         _LOGGER.debug("Inotify handle is (%d).", self.__inotify_fd)
 
-        self.__epoll = select.epoll()
-        self.__epoll.register(self.__inotify_fd, select.POLLIN)
+        self.__poll = select.poll()
```

Comment:
A comment here for the switch would be nice

Reply:
Good idea